apps/admin/src/features/system/perms/components/BaseModal.tsx

```diff
@@ -0,0 +1,59 @@
+import type { ScopeSubmitDto, ScopeTypeString } from '@raipiot-2f/api'
+
+import { useScopeConfigContext } from '../contexts'
+import { useScopeSubmitMutation } from '../mutations'
+
+interface BaseModalProps {
+  menuId: string
+}
+
+export function BaseModal({ menuId }: BaseModalProps) {
+  const { modal, form, formItems } = useScopeConfigContext()
+  const { isPending, mutate } = useScopeSubmitMutation()
+  const { type = 'api' } = useSearch({
+    from: '/_base/system/perms/$id'
+  }) as { type: ScopeTypeString }
```

Comment:
```suggestion
  })
```
validateSearch 添加 schema